<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>960</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPlainTextEdit" name="gobLapTextArea_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>288</width>
      <height>112</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell Condensed</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(43, 43, 43);
color: rgb(85, 255, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="undoRedoEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="centerOnScroll">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="gobLCD_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>576</width>
      <height>135</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell</family>
      <pointsize>48</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>150</y>
      <width>288</width>
      <height>97</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>       min     :      sec      :     1/100    </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="clearLapsButton_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>380</y>
      <width>191</width>
      <height>42</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Clear laps</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>320</y>
      <width>187</width>
      <height>42</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Start</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/img/img/Go.png</normaloff>:/img/img/Go.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="flagButton_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>320</y>
      <width>181</width>
      <height>42</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Lap</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/img/img/lap.png</normaloff>:/img/img/lap.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>380</y>
      <width>191</width>
      <height>42</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Reset</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/img/img/reset.png</normaloff>:/img/img/reset.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_chro">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>60</y>
      <width>201</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
